apps/examples/testcase/le_tc/kernel/tc_libc_timer.c

```diff
@@ -301,117 +301,177 @@ static void tc_libc_timer_mktime(void)
 }
 
 /**
-* @fn                   :tc_libc_timer_strftime
-* @brief                :Format time as string
-* @Scenario             :Copies into Source pointer the content of format, expanding its format specifiers
-*                        into the corresponding values that represent the time described in timeptr structure,
-*                        with a limit of maxsize characters.
-* API's covered         :strftime
-* Preconditions         :none
-* Postconditions        :none
-* @return               :void
+* @fn             :tc_libc_timer_strftime_strptime
+* @brief          :strftime - Format time as string
+*                  strptime - Format string as time
+* @Scenario       :strftime - Copies into Source pointer the content of format, expanding
+*                  its format specifiers into the corresponding values that represent the
+*                  time described in timeptr structure, with a limit of maxsize characters.
+*                  strptime - converts time string as the time described in timeptr
+*                  structure as per the format specifiers.
+* API's covered   :strftime, strptime
+* Preconditions   :none
+* Postconditions  :none
+* @return         :void
 */
-static void tc_libc_timer_strftime(void)
+static void tc_libc_timer_strftime_strptime(void)
```

Comment:
Why both are clubbed here ? we can create separate APIs right as both are used for different purpose ?

Reply:
`strftime` - convert a time tm structure to a time string.
`strptime` - convert a time string to a time tm structure.
as these two functions are inverse to each other, we can reduce code repeatability in test cases by combining them and also it is easy to verify input & output of both the functions.

Reply:
In order to maintain code uniformity, I will separate tc for strptime().